be1-go/hub/standard_hub/mod.go

```diff
@@ -92,6 +92,9 @@ type Hub struct {
 	// rootInbox and queries are used to help servers catchup to each other
 	rootInbox inbox.Inbox
 	queries   queries
+
+	// isCheckingOrganizerKey allows to enable or disable the security that allow only the organizer the LAO creation
```

Comment:
maybe server owner is better, because if it is false everyone can be organizers

Reply:
yes, you're right I will rename it to isCheckingOwnerKey